src/test/java/io/tiledb/spark/NullableAttributesTest.java

```diff
@@ -467,7 +467,7 @@ public void sparseWriteTest() throws Exception {
         .option("schema.dim.0.min", 1)
         .option("schema.dim.0.max", 8)
         .option("schema.dim.0.extent", 2)
-        .option("schema.attr.a1.filter_list", "(byteshuffle, -1), (gzip, -10)")
+        .option("schema.attr.a1.filter_list", "(scale_float, 1.0, 1.0, 8)")
```

Comment:
Can we parametrize this test over several filter types rather than dropping the old one?

Reply:
I introduced a new class to test all filters. I did not use the junit parameterized feature but used a simple for loop for simplicity. I will revisit all testing across all java repos when time allows. It is in my todos list.